### PR TITLE
Make btdb plugin python2 & python3 compatible

### PR DESCRIPTION
**Pull request by ngosang**
So the files are the same. It's easy to maintain, like other plugins.
No need to increase version, is only cosmetic.

### CODE REVIEW

File: nova/engines/btdb.py [hunk 1]
```diff
@@ -26,7 +26,13 @@
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 
-from HTMLParser import HTMLParser
+try:
```
Comment (Chocobo1) [inline nova/engines/btdb.py]: What about this style? The intention is a bit more clear IMO:
```python
import sys
if (sys.version_info > (3, 0)):
    # Python 3 imports in this block
else:
    # Python 2 imports in this block
```

Comment (Chocobo1) [reply nova/engines/btdb.py]: Also you can merge after you changed it (or not), I believe you've made sure it is compatiable to both py versions.

Comment (ngosang) [reply nova/engines/btdb.py]: @Chocobo1 ~~I like the idea but I prefer `if sys.version_info[0] >= 3:` and remove the #python3/2 comments.
I opened several PR, if you want, after everything is merged you can open a new PR with this change in all plugins and close this PR.~~

I don't know if all libraries are available from python  3.0. There are a lof of changes between python 3.0 and python 3.7...

Comment (Chocobo1) [reply nova/engines/btdb.py]: >I don't know if all libraries are available from python 3.0. There are a lof of changes between python 3.0 and python 3.7...


If you have time, you could trigger TravisCI to test it, it does test it with python 3.7 nightly.

**Comment by ngosang** [comment]
@Chocobo1 I'm merging this as is. Other plugins have the same format. We can change all plugins in the future, by now all is working fine.

**Pull request merged**